mne/source_estimate.py

```diff
@@ -3208,7 +3208,11 @@ def stc_near_sensors(evoked, trans, subject, distance=0.01, mode='sum',
     Returns
     -------
     stc : instance of SourceEstimate
-        The surface source estimate.
+        The surface source estimate. If using volume ``src``, ``stc``
+        will have the same number of vertices as the passed in
+        SourceSpace object. If using surface, then ``stc`` will
+        have the number of vertices on the ``pial`` surface that
+        get activated.
```

Comment:
This is actually not quite accurate.

If src is None, a surface source estimate will be produced, and the number of vertices will equal the number of pial-surface vertices that were close enough to the sensors to take on a non-zero volue.

If src is not None, a surface, volume, or mixed source estimate will be produced (depending on the kind of source space passed) and the vertices will match those of ``src`` (i.e., there may be me many all-zero values in ``stc.data``).

Reply:
Okay that makes more sense. I'll add that in.